apps/browser/src/autofill/content/content-message-handler.ts

```diff
@@ -37,6 +39,17 @@ async function handleAuthResultMessage(data: ContentMessageWindowData, referrer:
   await chrome.runtime.sendMessage({ command, code, state, lastpass, referrer });
 }
 
+/**
+ * Handles the Duo 2FA result message from the window.
+ *
+ * @param data - Data from the window message
+ * @param referrer - The referrer of the window
+ */
+async function handleDuoResultMessage(data: ContentMessageWindowData, referrer: string) {
+  const { command } = data;
+  await chrome.runtime.sendMessage({ command, code: data.code, referrer });
```

Comment:
Nit - You could also de-structure `code` here.

```suggestion
  const { command, code } = data;
  await chrome.runtime.sendMessage({ command, code:code, referrer });
```

Reply:
Also, can you please add a jest tests for this new method? Take a look at how I've handled calling the handler within the spec file for this.

Reply:
I didn't realize there were tests for this file! Awesome! Added!